github.ps1

```
@@ -158,9 +258,66 @@ function Publish-Release {
+        Write-Host "No assets found for pattern $pattern and for tag $tag"
+        return
+    }
+
+    if ($null -eq $assets.Count) {$count = "1 asset"} else {$count = "$($assets.Count) assets"}
+    Write-Host "$count found for pattern $pattern and for tag $tag!"
+    $asset = $assets | Select-Object -First 1
+    $url = $asset | Select-Object -ExpandProperty browser_download_url 
+
+    Write-Host "Downloading $($asset.name) from $url ..."
+	$filename = $url.Split('/')[-1]
++	Invoke-WebRequest -Uri $url -OutFile $filename
```

PSScriptAnalyzer (1.25.0)

[error] Missing expression after unary operator '+'.

(MissingExpressionAfterOperator)